nova/api/openstack/compute/sap_admin_api.py

```diff
@@ -69,6 +72,46 @@ def in_cluster_vmotion(self, req, body):
             common.raise_http_conflict_for_instance_invalid_state(
                 state_error, 'in_cluster_vmotion', server_id)
 
+    @_register_endpoint('GET')
+    def usage(self, req):
+        """Call
+        """
+        context = req.environ['nova.context']
+        context.can(sap_policies.POLICY_ROOT % 'usage:list')
+        return self._usage(context)
+
+    def show_usage(self, req, id):
+        """Call
+        """
```

Comment:
You forgot to fill in the method docs :)

Reply:
Fixed it.